### ISSUE TITLE
Ps button not workong as intended

### ISSUE THREAD

**Issue by jakerockz97** [MERLev/reVita#3]
I tried every button to map PS Button but I it seems to be not working.

Like, I wanted to exit the game and go to livescreen sticker like what PS button normally does but it doesn't do that. I already turn on the system button hack and tried toggling propagate and such, no luck so far. 

Please fix it!

**Comment by MERLev** [comment]
Yeah, this is unfortunate limitation. Both power and Sony buttons are "spaecial" ones, and are handled by system in a different way.  So, simulating them is not possible for now - I'll look into adding this in the future.

**Comment by pmsobrado** [comment]
Can I remap the PS button to something else? I tried but the button stills makes the current app to exit to the home :(
